PWGMM/Lumi/Tasks/lumi.cxx

```diff
@@ -10,90 +10,93 @@
-                                     }};
+  Configurable<uint64_t> ftts{"ftts", 1530319778000,
+                              "First time of time stamp"};
+  Configurable<int> nContribMax{"nContribMax", 2500,
+                                "Maximum number of contributors"};
+  Configurable<int> nContribMin{"nContribMin", 10,
+                                "Minimum number of contributors"};
+
+  HistogramRegistry histos{
+      "histos",
+      {
+          {"vertexx", "", {HistType::kTH1F, {{1000, -1, 1, "x"}}}},     //
+          {"vertexy", "", {HistType::kTH1F, {{1000, -1, 1, "y"}}}},     //
+          {"timestamp", "", {HistType::kTH1F, {{20000, 0, 2e7, "t"}}}}, //
+          {"vertexx_timestamp",
+           "",
+           {HistType::kTH2F, {{1000, 0, 4e6, "t"}, {2000, -1, 1, "x"}}}}, //
+          {"vertexy_timestamp",
+           "",
+           {HistType::kTH2F, {{1000, 0, 4e6, "t"}, {2000, -1, 1, "y"}}}},     //
+          {"chisquare", "", {HistType::kTH1F, {{1000, 0, 100, "#chi^{2}"}}}}, //
+
+          {"vertexx_Refitted", "", {HistType::kTH1F, {{1000, -1, 1, "x"}}}}, //
+          {"vertexy_Refitted", "", {HistType::kTH1F, {{1000, -1, 1, "y"}}}}, //
+          {"vertexx_Refitted_timestamp",
+           "",
+           {HistType::kTH2F, {{1000, 0, 4e6, "t"}, {2000, -1, 1, "x"}}}}, //
+          {"vertexy_Refitted_timestamp",
+           "",
+           {HistType::kTH2F, {{1000, 0, 4e6, "t"}, {2000, -1, 1, "y"}}}}, //
+          {"chisquare_Refitted",
+           "",
+           {HistType::kTH1F, {{1000, 0, 100, "#chi^{2}"}}}}, //
+
+          {"vertexx_Refitted_vertexx",
+           "",
+           {HistType::kTH2F, {{1000, -1, 1, "x"}, {1000, -1, 1, "rx"}}}}, //
+          {"vertexy_Refitted_vertexy",
+           "",
+           {HistType::kTH2F, {{1000, -1, 1, "y"}, {1000, -1, 1, "ry"}}}} //
+      }};
   bool doPVrefit = true;
 
-  void init(InitContext&)
-  {
-    ccdb->setURL(ccdburl);
-    ccdb->setCaching(true);
-    ccdb->setLocalObjectValidityChecking();
-    lut = o2::base::MatLayerCylSet::rectifyPtrFromFile(ccdb->get<o2::base::MatLayerCylSet>(ccdbpath_lut));
-    if (!o2::base::GeometryManager::isGeometryLoaded()) {
-      ccdb->get<TGeoManager>(ccdbpath_geo);
-    }
-    mRunNumber = 0;
-  }
-
-  void process(aod::Collision const& collision,
-               aod::BCsWithTimestamps const&,
-               o2::soa::Join<o2::aod::Tracks, o2::aod::TrackSelection, o2::aod::TracksCov, o2::aod::TracksExtra, o2::aod::TracksDCA> const& tracks,
-               o2::soa::Join<o2::aod::Tracks, o2::aod::TracksCov, o2::aod::TracksExtra> const& unfiltered_tracks)
-  {
+  void process(aod::Collision const &collision, aod::BCsWithTimestamps const &,
+               o2::soa::Join<o2::aod::Tracks, o2::aod::TrackSelection,
+                             o2::aod::TracksCov, o2::aod::TracksExtra,
+                             o2::aod::TracksExtended> const &tracks,
```

Comment:
The previous version of the task had correct `TracksDCA` in place, `TracksExtended` no longer exists.